tests/test_find.py

```diff
@@ -34,8 +34,8 @@
 
 from tuna.sql import DbCursor
 from tuna.utils.logger import setup_logger
-from tuna.find_db import FindDB
-from tuna.db_tables import ConvConfig, Job
+#from tuna.find_db import FindDBMixin
+#from tuna.db_tables import ConvConfig, Job
```

Comment:
why are these commented out? if pylint is complaining, then remove

Reply:
I believe test_find requires some modifications, seems like there is a variable name change in find_db script

Reply:
then that should have been fixed by whoever made that change. Please pull in the latest develop, uncomment this and make sure it passes pylint. Our CI indicated this is not an issue.